crates/matrix-sdk/src/room/mod.rs

```diff
@@ -2976,6 +2979,60 @@ impl Room {
             .await?;
         Ok(())
     }
+
+    /// Subscribe to live location sharing events for this room.
+    ///
+    /// The returned receiver will receive a new event for each sync response
+    /// that contains a 'm.beacon' event.
+    pub fn subscribe_to_live_location_shares(
+        &self,
+    ) -> (JoinHandle<()>, broadcast::Receiver<LiveLocationShare>) {
+        let (sender, receiver) = broadcast::channel(16);
```

Comment:
Why 16?

Reply:
I was following `subscribe_to_typing_notifications` as an example and it has 16

Reply:
OK, but I suspect we can have a lot more updates in case of live location sharing. I advice to increase the channel's size to 128 for example. Thoughts?

It's fine to get a small channel however, the consumer just need to handle the lag itself. If the lag happens often, it becomes annoying.